app/views/users/edit.html.erb

```diff
@@ -0,0 +1,30 @@
+<body class="signup">
+<div class="col-6 mx-auto">
+    <div class="jumbotron opacity">
+        <h1 class="text-center m-4">Update Your Profile</h1>
+        <%= form_for (@user) do |f| %>
+            <div class="form-group ">
+                <%= f.label :name %>
+                <%= f.text_field :name, class: "form-control" %>
+            </div>
+
+            <div class="form-group">
+                <%= f.label :email%>
+                <%= f.email_field :email, class: "form-control"%>
+            </div>
+            <div class="form-group">
+                <%= f.label :password%>
+                <%= f.password_field :password, class: "form-control"%>
+            </div>
+            <div class="form-group mb-4">
+                <%= f.label :password_confirmation%>
+                <%= f.password_field :password_confirmation, class: "form-control"%>
+            </div>
+            <div class="form-group ">
+                <%= f.label :avatar %>
+                <%= f.file_field :avatar, accept: "image/jpeg, image/git, image/png", class: "form-control" %>
+            </div>
+            <%= f.submit "Edit acount", class: "btn btn-primary btn-block"%>
+        <% end %>
```

Comment:
Please use the _form.html.erb to render your edit form. No need to create another form.